### ISSUE TITLE
GCN coin support

### ISSUE THREAD

**Issue by znunez** [walletgeneratornet/WalletGenerator.net#134]
This project does not seem to have support for GCN coin, yet this site has a GCN generator ( [http://www.gcn.press/](http://www.gcn.press/) ) am I missing something or possibly they are using a fork?

**Comment by MichaelMure** [comment]
Yes, it's a fork. They could have contributed the support back, but it's allowed by the license.
